app/src/main/java/pm/gnosis/heimdall/ui/two_factor/Select2FaActivity.kt

```diff
@@ -62,10 +99,12 @@ open class Select2FaActivity : NfcActivity() {
         startActivityForResult(intent, AUTHENTICATOR_REQUEST_CODE)
     }
 
-    private fun onSelected(type: AuthenticatorInfo.Type) {
+    protected fun onSelected(type: AuthenticatorInfo.Type) {
         selectedAuthenticator = type
```

Comment:
Would it be better to make `selectedAuthenticator` nullable and have it as `null` by default (since it can be "nothing" now)

Reply:
sure. It also makes sense to select nothing (leave radio button unchecked) if NFC is not available.